api/src/main/java/uk/ac/ox/oucs/vle/CourseSignup.java

```diff
@@ -46,9 +46,9 @@
 	
 	/**
 	 * 
-	 * @return Text supporting signup
+	 * @return String array of student requirements/comments
 	 */
-	public String getNotes();
+	public String[] getNotes();
```

Comment:
Why not just create another method? Arrays are easy to get wrong in understand what's in each element without looking at documentation.